### PR TITLE
Translating Navigation of Tutorial

### PR DESCRIPTION
**Pull request by ChrisB9**

### CODE REVIEW

File: content/tutorial/nav.yml [hunk 1]
```diff
@@ -1,111 +1,111 @@
...
           href: /tutorial/tutorial.html#what-is-react
           forceInternal: true
         - id: inspecting-the-starter-code
-          title: Inspecting the Starter Code
+          title: Betrachten des ersten Codes
           href: /tutorial/tutorial.html#inspecting-the-starter-code
           forceInternal: true
         - id: passing-data-through-props
-          title: Passing Data Through Props
+          title: Daten über Properties weitergeben
```
Comment (ph1p) [inline content/tutorial/nav.yml]: I have also added "properties" to the list #10

File: content/tutorial/nav.yml [hunk 2]
```diff
@@ -1,111 +1,111 @@
...
           href: /tutorial/tutorial.html#lifting-state-up-again
           forceInternal: true
         - id: showing-the-past-moves
-          title: Showing the Past Moves
+          title: Die letzten Züge anzeigen
           href: /tutorial/tutorial.html#showing-the-past-moves
           forceInternal: true
         - id: picking-a-key
-          title: Picking a Key
+          title: Ein key auswählen
```
Comment (ph1p) [inline content/tutorial/nav.yml]:
```suggestion
          title: Einen Schlüssel (key) auswählen
```

File: content/tutorial/nav.yml [hunk 3]
```diff
@@ -1,111 +1,111 @@
...
           href: /tutorial/tutorial.html#picking-a-key
           forceInternal: true
         - id: implementing-time-travel
-          title: Implementing Time Travel
+          title: Zeitreisen implementieren
           href: /tutorial/tutorial.html#implementing-time-travel
           forceInternal: true
         - id: wrapping-up
-          title: Wrapping Up
+          title: Zusammenfassen
```
Comment (ph1p) [inline content/tutorial/nav.yml]:
```suggestion
          title: Zusammenfassung
```

```suggestion
          title: Abschluss
```

File: content/tutorial/nav.yml [hunk 4]
```diff
@@ -1,111 +1,111 @@
...
           forceInternal: true
     - id: completing-the-game
-      title: Completing the Game
+      title: Das Spiel fertigstellen
       href: /tutorial/tutorial.html#completing-the-game
       forceInternal: true
       subitems:
         - id: lifting-state-up
-          title: Lifting State Up
+          title: Den State zurückbringen
```
[the remark this answers is not in the document]
Comment (ChrisB9) [reply content/tutorial/nav.yml]: i really had troubles with that one. But you are right the more i think about it...

```suggestion
          title: Den State hoch holen
```

File: content/tutorial/nav.yml [hunk 5]
```diff
@@ -1,111 +1,111 @@
...
       forceInternal: true
       subitems:
         - id: storing-a-history-of-moves
-          title: Storing a History of Moves
+          title: Einen Zug-Verlauf speichern
           href: /tutorial/tutorial.html#storing-a-history-of-moves
           forceInternal: true
         - id: lifting-state-up-again
-          title: Lifting State Up, Again
+          title: Den State nochmal zurückbringen
```
[the remark this answers is not in the document]
Comment (ChrisB9) [reply content/tutorial/nav.yml]:
```suggestion
          title: Den State nochmal hoch holen
```

**Pull request merged**